javascript/app.js

```diff
@@ -0,0 +1,905 @@
+console.log(kristynsCloset); // Test
+// Put together an outfit for Thom! Using bracket notation, access the first element in Thom's shirts array.
+// In the same way, access one item from Thom's pants array.
+// Access one item from Thom's accessories array.
+// Log a sentence about what Thom's wearing. Example: "Thom is looking fierce in a grey button-up, jeans and wool scarf!"
+// I like making my pockito choose for me, gonna define a randomizer to avoid repetition
+const randomNumber = (limit) => {
+  return Math.floor(Math.random() * Math.floor(limit));
+}
+console.log(`Thom is looking fierce in a ${thomsCloset[0][randomNumber(thomsCloset[0].length)]}, ${thomsCloset[1][randomNumber(thomsCloset[1].length)]} and ${thomsCloset[2][randomNumber(thomsCloset[2].length)]}!`);
+// Get more specific about what kind of PJs Thom's wearing this winter. Modify the name of his PJ pants to Footie Pajamas.
+thomsCloset[1].splice(thomsCloset[1].indexOf("PJs"), 1, "Footie Pajamas");
+console.log(thomsCloset);
+// 🔴 Commit.
+
+
+
+// IV. Functions
+
+
+// A. printGreeting
+
+// Do you think you could write a function called printGreeting with a parameter name that returns a greeting with the argument interpolated into the greeting?
+// Like so?
+// console.log(printGreeting("Slimer"));
+// => Hello there, Slimer!
+// You think you could? I think so too. Feel free to skip this problem, because you've already done it. If you've done the problem twice, read entire problems carefully before doing them from now on.
+// I dont' mind redoing, good practice :)
+const printGreeting = (name) => {
+	return `Hello there, ${name}!`; 
+}
+console.log(printGreeting("Slimer"));
+// 🔴 Commit.
+
+
+// B. printCool
+
+// Write a function printCool that accepts one parameter, name as an argument. The function should print the name and a message saying that that person is cool.
+// console.log(printCool("Captain Reynolds"));
+// => "Captain Reynolds is cool";
+const printCool = (name) => {
+	return `${name} is hella cool`; 
+}
+console.log(printCool("Hela"));
+// 🔴 Commit.
+
+
+// C. calculateCube
+
+// Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+// console.log(calculateCube(5));
+// => 125
+const calculateCube = (number) => {
+	// Use Matgh.pow(a, b) which is a^b
+	return `${Math.pow(number, 3)} is the cube of ${number}`;
+}
+console.log(calculateCube(5));
+// 🔴 Commit.
+
+
+// D. isVowel
+// Write a function isVowel that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise. The vowel could be upper or lower case.
+// console.log(isVowel("a"));
+// => true
+const isVowel = (string) => {
+	// Make string toLowerCase to avoid condition repetition overload
+	string.toLowerCase();
+	if (string.length === 1 && string === "a" || string === "e" || string === "i" || string === "o" || string === "u" || string === "y") {
```

Comment:
technically this isn't working the way you envision...
first condition is length == 1 and string == a
second condition is string == e
third condition is string == i.....
to get your logic to be working the way i think you intended, use an extra set of parentheses around the  vowel comparisons
like so:
```
if (string.length === 1 && (string === "a" || string === "e" || string === "i" || string === "o" || string === "u" || string === "y") ) 
     

```